src/OrbitQt/orbitmainwindow.h

```diff
@@ -132,6 +132,11 @@ class OrbitMainWindow final : public QMainWindow, public orbit_gl::MainWindowInt
   orbit_base::CanceledOr<void> DisplayStopDownloadDialog(
       const orbit_client_data::ModuleData* module) override;
 
+  void SetTopDownInspection(std::unique_ptr<CallTreeView> top_down_view) override;
+  void SetBottomUpInspection(std::unique_ptr<CallTreeView> bottom_up_view) override;
```

Comment:
I'm wondering if these should be a single method as they are always called together. Basically I'm thinking about the parallel with `ClearInspection`, which is a single method that clears both. But I'm not sure.

Reply:
I originally had this and don't have a strong preference; the only reason for keeping them separate is because I will be adding more parameters for this when I am also implementing the samples tab in the same way and it may be a lot, but they should always be used with each other so I'm fine with combing them. Done.